inductiva/coastal/README.md

```diff
@@ -8,10 +8,76 @@ is performed using the [SWASH](https://swash.sourceforge.io/) simulator.
 
 ### Example
 
-Start by creating or loading a bathymetry. This can be done e.g. by reading an
+Note: You can follow this example in a notebook or run the script
+`random_bathymetry_demo.py` available in this folder, that contains all the code
+snippets exhibited here. Don't forget to insert your API key for the script to work!
+
+The bathymetry that we use can be loaded from a real scenario or randomly generated
+by us. To avoid having to download any data for now, we will start by run a simulation
+on synthetic bathymetry , that is procedurally generated using the API.
+
+We start by generating a random bathymetry.
+```python
+import inductiva
+
+bathymetry = inductiva.coastal.Bathymetry.from_random_depths(
+        x_range=(0, 100),
+        y_range=(0, 100),
+        x_num=100,
+        y_num=100,
+        max_depth=50)
+```
+
+This bathymetry is defined in a 50x50 grid, that represent a domain 100m
```

Comment:
This can be a comment. Moreover, I do not see why this is a 50x50 grid. The code is readable and explained well. Just add a top comment to mention the units and that is it.